### DISCUSSION TITLE
Pass data to iframe

### DISCUSSION THREAD

**Discussion by niosme** [biati-digital/glightbox#453] [Q&A]
Hello! How can i pass some data to an iframe?
Is it possible inside the url and then get them from the other side?
What is the steps to do it?
E.g.

data-from='homepage'

so we now the iframe opened from the homepage when we open the iframe

**Comment by gingerchew** [comment]
I'm pretty sure it's not possible, but what is your use case?

**Reply by niosme** [reply]
I have a confirm address php file which i call with iframe popup.I am doing now with colorbox which you can pass the data when you open an iframe and get them from the other side.

The think is that the main functions are the same but it have differences when you call it from homepage and when you call it from another view page. From the another view page it also auto completes with javascript some fields they dont exist in homepage and doing some work it doesnt needed to be done in the homepage.

Maybe i can check in which page i am by querySelector a class exists in this page and check if i get null or not as a result which it may work i think.Like looking for unique classes in that page :p
